CMakeLists.txt

```diff
@@ -1144,6 +1170,8 @@ function(add_library_variant target_arch)
             "${lit_test_executor}"
             "${LLVM_TOOLCHAIN_C_LIBRARY}_${variant}-install"
         )
+        set(enable_exceptions OFF)
```

Comment:
It's a neat trick to use dynamic scope to pass these parameters into `add_libcxx_libcxxabi_libunwind` without having to include them in the function's parameter list, but I'm not sure it improves readability. I think it would be better to make them function parameters in the usual way, so that callers _know_ they have to think about which way to set them.

This means that in our downstream code that also calls that function, we'll have to make a corresponding change, but I think that's the best thing in the long run.